resources/puppetlabs/lein-ezbake/template/foss/ext/debian/postinst.erb

```diff
@@ -4,12 +4,16 @@
 
 # On upgrade, we should restart the service if it's running
 if [ $1 = 'configure' -a -n $2 ] ; then
-  if [ -x '/etc/init.d/<%= EZBake::Config[:project] %>' ] ; then
-    # Using sysv
-    invoke-rc.d <%= EZBake::Config[:project] %> try-restart || :
-  elif [ -x /bin/systemctl ] ; then
+  if [ -e "/lib/systemd/system/<%= EZBake::Config[:project] -%>.service" ] ; then
     # Using systemd
+    if [ -e '/etc/init.d/<%= EZBake::Config[:project] %>' ] ; then
+      # We need to clean up the old sysv files if they still exist
+      rm '/etc/init.d/<%= EZBake::Config[:project] %>'
```

Comment:
So with both in the package, this will also have the downside of nuking the sysv script from all deb packages in the postinstall.